kernel/arch/dreamcast/include/arch/arch.h

```diff
@@ -21,7 +21,7 @@
 #include <kos/cdefs.h>
 __BEGIN_DECLS
 
-#include <dc/video.h>
```

Comment:
This was mysterious. All I can guess is that some time in the distant past it was needed. At this point it was just a roundabout way to get to arch/types.